pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```

Comment:
is this expected?

Reply:
Nope, not sure how this got updated. I'll revert this file.